app/components/c-add-to-cart.md

```diff
@@ -0,0 +1,11 @@
+Renders add to cart component.
```

Comment:
- [ ] Please move description to top of `export default {}` of component itself.